skills/unity-vrc-world-sdk-3/references/lighting.md

```
@@ -210,7 +210,7 @@ Recommended shaders:
 ### Blurry Lightmaps
 
 **Solution**:
-```
+```text
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)